### PR TITLE
Created a static method to read the props from yaml file

### PR DESCRIPTION
**Pull request by shridharkalagi**
Read the file once and use it everywhere. Also file reading is limited to a single class now

### CODE REVIEW

File: src/main/java/com/nakal/utils/YamlReader.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package com.nakal.utils;
+
+import org.yaml.snakeyaml.Yaml;
```
Comment (SrinivasanTarget) [inline src/main/java/com/nakal/utils/YamlReader.java:3]: where did we get this from? I don't any dependencies been added here. Is it internal dep of something else.

Comment (saikrishna321) [reply src/main/java/com/nakal/utils/YamlReader.java:3]: snakeyaml dependency is already part of project

Comment (SrinivasanTarget) [reply src/main/java/com/nakal/utils/YamlReader.java:3]: oh cool

Comment (shridharkalagi) [reply src/main/java/com/nakal/utils/YamlReader.java:3]: Can this be merged?

**Pull request merged**